go.mod

```diff
@@ -40,4 +41,5 @@ replace (
 	k8s.io/api => k8s.io/api v0.17.4 // Required by client-go
 	k8s.io/apimachinery => k8s.io/apimachinery v0.17.4 // Required by client-go
 	k8s.io/client-go => k8s.io/client-go v0.17.4 // Required by prometheus-operator
+	sigs.k8s.io/controller-runtime => sigs.k8s.io/controller-runtime v0.5.2
```

Comment:
why do we need replace for controller-runtime?

Reply:
v0.6.1 is required by `sigs.k8s.io/mcs-api`, but all other packages use v0.5.2 and don't support v0.6.1

Reply:
can we please update operator-sdk instead? in a separate PR

Reply:
can we please merge https://github.com/percona/percona-server-mongodb-operator/pull/891 first, and rebase this PR after the merge

Reply:
@pooknull @delgod I believe we need to merge #860 rather than #891. @pooknull Could you check it out to see if it works for this PR?

Reply:
@egegunes #860 works for this PR. I'm closing #891